### PR TITLE
Build conda linux for domains. Run smoke tests in separate conda environment in Linux

### PR DESCRIPTION
**Pull request by atalman**
Build conda Linux for domains. Run smoke tests in separate conda environment

### CODE REVIEW

**Review by weiwangmeta — APPROVED** [review]
Looks great!

File: .github/workflows/build_conda_linux.yml [hunk 1]
```diff
@@ -164,31 +164,36 @@ jobs:
...
           if [[ "${{ inputs.package-name }}" = "torchvision" ]]; then
-              ${CONDA_RUN} conda install -y 'numpy>=1.11'
-              ${CONDA_RUN} conda install pillow
+              ${CONDA_RUN_SMOKE} conda install -y 'numpy>=1.11'
+              ${CONDA_RUN_SMOKE} conda install pillow
           fi
           if [[ "${{ inputs.package-name }}" = "torchdata" ]]; then
-              ${CONDA_RUN} conda install -y portalocker>=2.0.0
+              ${CONDA_RUN_SMOKE} conda install -y portalocker>=2.0.0
           fi
```
Comment (malfet) [inline .github/workflows/build_conda_linux.yml:177]: Sorry, but why this is needed?

File: .github/workflows/build_conda_linux.yml [hunk 2]
```diff
@@ -164,31 +164,36 @@ jobs:
...
 
           if [[ "${GPU_ARCH}" = "cuda" ]]; then
-            ${CONDA_RUN} conda install -v -y -c pytorch-"${CHANNEL}" -c nvidia pytorch pytorch-cuda="${GPU_ARCH_VERSION}"
+            ${CONDA_RUN_SMOKE} conda install -v -y -c pytorch-"${CHANNEL}" -c nvidia pytorch pytorch-cuda="${GPU_ARCH_VERSION}"
           else
-            ${CONDA_RUN} conda install -v -y -c pytorch-"${CHANNEL}" pytorch cpuonly
+            ${CONDA_RUN_SMOKE} conda install -v -y -c pytorch-"${CHANNEL}" pytorch cpuonly
           fi
+          ${CONDA_RUN_SMOKE} conda install  -v -y -c file://"${PWD}/${CONDA_OUTPUT_PATH}" "${{ inputs.package-name }}"
```
Comment (malfet) [inline .github/workflows/build_conda_linux.yml:184]:
```suggestion
          ${CONDA_RUN_SMOKE} conda install  -v -y -c pytorch-"${CHANNEL}" --use-local  "${PWD}/${CONDA_OUTPUT_PATH}/${{ inputs.package-name }}*"
```

**Review by malfet — COMMENTED** [review]
Let's figure out how to use `--use-local` rather than explicitly specify the dependencies

File: .github/workflows/build_conda_linux.yml (hunk 2 above)
Comment (atalman) [reply .github/workflows/build_conda_linux.yml:184]: I tried this but it was not working, can try again if you like

File: .github/workflows/build_conda_linux.yml (hunk 1 above)
Comment (atalman) [reply .github/workflows/build_conda_linux.yml:177]: Not sure, I think this was required for when Nova was developed at some point. Maybe we can try to remove these and see what's up

**Pull request merged**